spec/lib/bank/account_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'bank'
+
+describe Bank::Account do
+
+  before :each do
+    @bank = Bank::Account.new("Checking", 50)
+  end
+
+  describe "#new" do
+    it "creates a new instance" do
+      expect(Bank::Account).to respond_to(:new)
```

Comment:
All Ruby classes inherit from Object, which defines `.new`, so you don't need to really test for it.